src/config/index.ts

```diff
@@ -5,9 +5,15 @@ const appName = envVariables.REACT_APP_APPLICATION_NAME;
 const API_PORT = envVariables.REACT_APP_GLIFIC_API_PORT;
 const PROTOCOL = window.location.protocol;
 const API_PREFIX = envVariables.REACT_APP_API_PREFIX;
-const HOSTNAME = API_PREFIX
-  ? `${API_PREFIX}.${window.location.hostname}`
-  : window.location.hostname;
+// added support for localhost, need to update it after backend fixes
+let HOSTNAME;
+if (window.location.hostname === 'localhost') {
+  HOSTNAME = 'localhost';
+} else if (API_PREFIX) {
+  HOSTNAME = `${API_PREFIX}.${window.location.hostname}`;
+} else {
+  HOSTNAME = window.location.hostname;
+}
```

Comment:
I don't think these changes are needed. It will work for localhost if you don't add the API_PREFIX in the env.